packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -59,6 +57,20 @@ export interface HasPlaceFields {
      * Omit if `Tiebreak.Right` for terseness.
      */
     tiebreak?: Tiebreak;
+
+    /**
+     * Record of relevant information about changes this mark has been rebased over.
+     */
+    lineage?: LineageEvent[];
+}
+
+export interface LineageEvent {
+    readonly revision: RevisionTag;
+
+    /**
+     * The position of this mark within the range of detached nodes in this revision
```

Comment:
Is that acrosss all detached nodes for that field for that revision, or only the detached nodes that merged into the gap that the mark sporting this lienage targets? Would be good to clarify in the comment.